server/src/pages/shot/editor.js

```diff
@@ -51,7 +53,8 @@ exports.Editor = class Editor extends React.Component {
       tool: "pen",
       color: "#000",
       lineWidth: 5,
-      actionsDisabled: true
+      actionsDisabled: true,
+      colorName: "black"
```

Comment:
so nit: but perhaps move this line up next to the `color` one?

Reply:
+1